src/pages/DecisionTable/DecisionTable.tsx

```diff
@@ -37,12 +37,28 @@ const DecisionTable = () => {
   const [pageSize, setPageSize] = useState(10);
   const [total, setTotal] = useState<number>(0);
   const [count, setCount] = useState<number>(0);
-  
-  const handleDelete = (id: number) => {
-    const filteredData = data.filter(d => d.id !== id);
-    setData([...filteredData]);
-    setTotal(total-1);
-    setCount(count-1);
+
+  const fetchData = async () => {
+    let jwt = AuthStore.getToken() as string;
+    let decodedJwt = jwt_decode(jwt) as any;
+    let roles = decodedJwt['http://schemas.microsoft.com/ws/2008/06/identity/claims/role'] as string[];
```

Comment:
it is better to set such configuration strings as some const variables rather than hardcoding

Reply:
fixed